agent/node.go

```diff
@@ -104,14 +104,16 @@ func NewNode(c *NodeConfig) (*Node, error) {
 	}
 	stateFile := filepath.Join(c.StateDir, stateFilename)
 	dt, err := ioutil.ReadFile(stateFile)
-	var p []api.Peer
-	if err != nil && !os.IsNotExist(err) {
+	if err != nil {
+		if os.IsNotExist(err) {
+			err = fmt.Errorf("local state file does not exist")
```

Comment:
Could we not hide the error? use `errors.Wrap(err, "local state file does not exist")`